go/controller/internal/controller/scmconnection_controller.go

```diff
@@ -100,11 +100,6 @@ func (r *ScmConnectionReconciler) Reconcile(ctx context.Context, req reconcile.R
 	// Mark resource as managed by this operator.
 	utils.MarkCondition(scm.SetCondition, v1alpha1.ReadonlyConditionType, v1.ConditionFalse, v1alpha1.ReadonlyConditionReason, "")
 
-	err = r.tryAddControllerRef(ctx, scm)
```

Comment:
if there's no owner ref we still need a mechanism to propagate updates from the secret to the scm connection.  Usually this is done via a watches + annotation in other controllers.

Reply:
ahh, ok. I thought you wanted to keep it completely separate. I will update it.